pkgs/development/python-modules/openusd/default.nix

```diff
@@ -65,13 +66,12 @@ buildPythonPackage rec {
 
   outputs = [ "out" ] ++ lib.optional withDocs "doc";
 
-  format = "other";
-
   patches = [
     (fetchpatch {
       name = "port-to-embree-4.patch";
-      url = "https://github.com/PixarAnimationStudios/OpenUSD/pull/2266/commits/4b6c23d459c602fdac5e0ebc9b7722cbd5475e86.patch";
-      hash = "sha256-yjqdGAVqfEsOX1W/tG6c+GgQLYya5U9xgUe/sNIuDbw=";
+      # https://github.com/PixarAnimationStudios/OpenUSD/pull/2266
+      url = "https://github.com/PixarAnimationStudios/OpenUSD/commit/c8fec1342e05dca98a1afd4ea93c7a5f0b41e25b.patch?full_index=1";
```

Comment:
Is `full_index=1` needed?

Reply:
https://github.com/NixOS/nixpkgs/pull/351902#discussion_r1827042323
Is why `full_index=1` was added

Reply:
It seems like `full_index=1` just expands the commit hashes used in the diff. I don't think it's strictly needed, but at the same time it doesn't really hurt.